modules/combined/test/tests/sliding_block/sliding/constraint/frictional_02_penalty.i

```diff
@@ -176,7 +165,6 @@
 []
 
 [Outputs]
-  file_base = frictional_02_penalty_out
   interval = 10
   [./exodus]
```

Comment:
You could recover the original behavior of the file names by renaming the 'exodus' block to 'out'.  I don't like calling the files '*exodus.e'